unit_testing/cli/test_cli_sct_compute_ernst_angle.py

```diff
@@ -1,14 +1,17 @@
-from pytest_console_scripts import script_runner
 import pytest
 import logging
 
+from spinalcordtoolbox.scripts import sct_compute_ernst_angle
+
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.script_launch_mode('subprocess')
-def test_sct_compute_ernst_angle_backwards_compat(script_runner):
-    ret = script_runner.run('sct_testing', '--function', 'sct_compute_ernst_angle')
-    logger.debug(f"{ret.stdout}")
-    logger.debug(f"{ret.stderr}")
-    assert ret.success
-    assert ret.stderr == ''
+@pytest.mark.sct_testing
+@pytest.mark.usefixtures("run_in_sct_testing_data_dir")
+def test_sct_compute_ernst_angle_value_against_groundtruth():
+    """Run the CLI script and verify that computed ernst angle is equivalent to known ground truth value."""
+    fname_out = 'ernst_angle.txt'
+    sct_compute_ernst_angle.main(argv=['-tr', '2000', '-t1', '850', '-o', fname_out])
+    with open(fname_out, 'r') as f:
+        angle_result = float(f.read())
+    assert angle_result == pytest.approx(84.543553255)
```

Comment:
is this the same as `abs(angle_result - param_test.angle_gt) < param_test.threshold:`? approx considers numbers within a relative tolerance of 1e-6 (i.e. one part in a million) of its expected value to be equal, you threshold is bigger than that